lib/tasks/index.rake

```diff
@@ -30,7 +30,7 @@ task :log_indexer => :environment do |t, args|
   raise 'Log type must be preassembly, remediate or csv.' unless ['preassembly','remediate','csv'].include? log_type
   raise 'Log file not found.' unless File.readable? log_file_path
 
-  target_config=BaseIndexer.solr_configuration_class_name.constantize.instance.get_configuration_hash[target.upcase]
+  target_config = Settings.SOLR_TARGETS[target]
```

Comment:
My thought here is that there is no way to pass to a non-upcased configuration. The configuration should be the configuration. If we want all upcased targets, can't we just upcase them ourselves in the config?